### PR TITLE
fix(preview): If preview errors, set thumbnails sidebar as closed

### PR DESCRIPTION
**Pull request by ConradJChan**
This solves the issue where when the thumbnails sidebar is open by default, the `ContentPreview` prev file nav button is correctly positioned.

Also ensure the prev file nav button is moved back to the left in case of preview error.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: src/elements/content-preview/ContentPreview.js [hunk 1]
```diff
@@ -708,6 +712,12 @@ class ContentPreview extends PureComponent<Props, State> {
         this.preview.addListener('preview_metric', this.onPreviewMetric);
         this.preview.addListener('thumbnailsOpen', () => this.setState({ isThumbnailSidebarOpen: true }));
         this.preview.addListener('thumbnailsClose', () => this.setState({ isThumbnailSidebarOpen: false }));
+        this.preview.addListener('viewer', viewer => {
```
Comment (jstoffan) [inline src/elements/content-preview/ContentPreview.js]: Could we use the `thumbnailsOpen` and `thumbnailsClosed` events for this?

**Pull request merged**